docs/rp2/quickref.rst

```diff
@@ -84,19 +96,35 @@ Use the :ref:`machine.Pin <machine.Pin>` class::
 UART (serial bus)
 -----------------
 
-See :ref:`machine.UART <machine.UART>`. ::
+There are two UARTs, UART0 and UIART1. UART0 can be mapped to GPIO 1/2, 12/13
+and 16/17, and UART1 to GPIO 6/7 and 8/9. Since for using the UART GPIO numbers
+have to be used, these should be mentioned and can be seen in the pin diagram
+below:
+
+.. image:: img/UARTpinout.png
+    :alt: Pin Diagram Raspberry Pi Pico
+    :width: 640px
+
 
-    from machine import UART
 
-    uart1 = UART(1, baudrate=9600, tx=33, rx=32)
+See :ref:`machine.UART <machine.UART>`. ::
+
+    from machine import UART, Pin
+    uart1 = UART(1, baudrate=9600, tx=Pin(4), rx=Pin(5))
     uart1.write('hello')  # write 5 bytes
     uart1.read(5)         # read up to 5 bytes
 
+.. note::
+ 	
+	REPL over UART is disabled by default. You can see the :ref:`rp2_intro` for
+	details on how to enable REPL over UART.
+
 
 PWM (pulse width modulation)
 ----------------------------
 
-How does PWM work on the RPi RP2xxx?
+There are 8 independent channels each of which have two outputs making it 16 PWM
```

Comment:
If I'm being super-picky the old-school rule is to write the words for numbers less than ten, digits for 10 and above. The more modern take is to just write digits.

I'm not rigid on either rule but there's '8' and 'two' in this one sentence - which is a little inconsistent either way!